cli_ui/__init__.py

```diff
@@ -355,6 +355,47 @@ def info_progress(prefix: str, value: float, max_value: float) -> None:
         sys.stdout.flush()
 
 
+def info_progress_bar(
+    prefix: str, value: float, max_value: float, fileObj: FileObj = sys.stdout
+) -> None:
+    """ Display info progress as a bar.
+
+    :param value: the current value
+    :param max_value: the max value
+    :param prefix: the prefix message to print
+
+
+    """
+    MAX_SIZE = 10
+    BLOCK_CHAR = "█"
+    COLOR_LIST = [
+        colorama.Fore.RED + colorama.Style.BRIGHT,
+        colorama.Fore.RED + colorama.Style.BRIGHT,
+        colorama.Fore.RED + colorama.Style.NORMAL,
+        colorama.Fore.RED + colorama.Style.DIM,
+        colorama.Fore.GREEN + colorama.Style.DIM,
+        colorama.Fore.GREEN + colorama.Style.NORMAL,
+        colorama.Fore.GREEN + colorama.Style.BRIGHT,
+        colorama.Fore.GREEN + colorama.Style.BRIGHT,
+        colorama.Fore.GREEN + colorama.Style.BRIGHT,
+    ]
+    COLOR_RESET = colorama.Fore.WHITE + colorama.Style.NORMAL
+
+    percent = float(value) / max_value * 100
+    color = COLOR_LIST[int(percent / 12.5)]
```

Comment:
* this won' t work if percent is bigger than 100
* not sure why yo go through a percentage anyway. You can just use ration =value / max_value
* don' t hard-code 12.5 - use len(COLOR_LIST) instead